PeepPeep/PeepPeep/View/InitialSetupScreen/InitialSetupView.swift

```diff
@@ -4,16 +4,38 @@
 //
 //  Created by Ha Jong Myeong on 2023/07/12.
 //
+
 import SwiftUI
+import DeviceActivity
+import FamilyControls
 
 struct InitialSetupView: View {
+    @State var show = false
```

Comment:
Bool의 property는 꼭 의문형으로 하는걸 추천해요 ! 
그렇게되면 값을 보자마자 Bool의 타입의 유추가 가능해요 :)

```suggestion
    @State var isNeededToShow = false // or isShowing 
```

Reply:
작은 부분까지 캐치 해주셔서 감사해요 🥹